### ISSUE TITLE
Missing information on comments made on pages

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#510]
We have discovered that some idiots have been putting problem reports into the comments function, and we're trying to clear through those. This is OK when they have commented on sources, but we also have a load of page comments but cannot trace the relevant page because the link back is 'not implemented' in the comment. Could you possibly implement the link as a matter of some urgency please?

**Comment by ahankinson** [comment]
Is this related to https://github.com/diamm/diamm/issues/469?

**Comment by juliacmcf** [comment]
Yes — we’re trying to trace the comments that should be problem reports back to their origins, but there’s currently no way to do it with page comments. The source comments have a nice link so we know what they are attached to.

**Comment by ahankinson** [comment]
Hmmm... There is no way of entering page comments in the interface, so if you're seeing the data in there it's from the old site, not the new one. I migrated it because I didn't want to lose that data, but I'm not sure if it's even 'hooked up'...

**Comment by juliacmcf** [comment]
OK, that will have to be fine. It means we’ve lost the information in the corrections, but it’s probably not the end of the world. It’s not data I had in filemaker as it was only on the website, so there’s no way to look it up retrospectively. It will save us considerable work not having to deal with these!
J
